python/ray/tune/tests/test_sync.py

```diff
@@ -35,20 +35,6 @@ def tearDown(self):
         ray.shutdown()
         _register_all()  # re-register the evicted objects
 
-    @patch("ray.tune.sync_client.S3_PREFIX", "test")
```

Comment:
If we don't specify an upload dir, we automatically disable cloud checkpointing. Thus, we can't expect an error to be raised.
Previously the exception was raised because we used `sync_to_cloud` without an `upload_dir` - however, `sync_to_cloud` does not exist anymore.